### PR TITLE
Perform some fixes and rewordings in the "Getting started" documentation

### PR DESCRIPTION
**Pull request by Lonnytunes**
Rewordings are proposals. Feel free to refuse ones you don't like.

### CODE REVIEW

File: docs/content/getting-started/basics.md [hunk 1]
```diff
@@ -5,64 +5,66 @@ cases that it addresses:
...
-It will list you all the backups present on your disk, choose the one you want to restore!
+This command will list you all backups available on your disk. Choose the one
+you want to restore!
 
 These commands are only kinds of shortcut for `pg_dump`/`pg_restore` or
 `mysqldump`/`mysql` but much more handy to use. They find the correct binaries
 and execute them with the correct options for you.
 
-## Use case 2 - Anonymizing while getting back production data on your local environment
+## Use case 2 - Working with anonymized production data on your local environment
```
Comment (SimonMellerin) [inline docs/content/getting-started/basics.md:41]: Not sure about this one,  what I wanted to express was "How to anonymize sensitive data coming from your prod"
Here, with your suggestion, I understand that data are already anonymized.

File: docs/content/getting-started/basics.md [hunk 2]
```diff
@@ -5,64 +5,66 @@ cases that it addresses:
...
-you got back the file on your machine.
+Let's say you have launched a `console db-tools:backup` on your production environment
+and downloaded the backup file on your machine.
 
-You could run `console db-tools:restore` to populate your database with those production
-data. Doing so, you will end up with sensitive data on your machine: and you don't want that.
-First of all, because in most cases (for example in UE, with GDPR): that's illegal. Secondly because you just
-don't want to know personnal data from your (or your client's) customers.
+You could run `console db-tools:restore` to populate your database from the
+freshly downloaded backup file. But in doing so, you will end up with sensitive
```
Comment (SimonMellerin) [inline docs/content/getting-started/basics.md:50]: not sure about `but in doing so`, `but doing so` sounds better to me

File: docs/content/getting-started/basics.md [hunk 3]
```diff
@@ -86,52 +88,53 @@ class User
 }
 ```
 
-With this configuration, after you launch `console db-tools:anonymize`, all your user's email addresses
-will be gone and replace with hashed ones.
+With this configuration above, after you launched `console db-tools:anonymize`,
+all your user's email addresses will be gone and replaced with hashed ones.
```
Comment (SimonMellerin) [inline docs/content/getting-started/basics.md:92]:
```suggestion
With the above configuration, after you launched `console db-tools:anonymize`,
all your user's email addresses will be gone and replaced with hashed ones.
```

I think we need to choose `this configuration` or `the above configuration` but not both ^^

**Review by SimonMellerin — CHANGES_REQUESTED** [review]
Thanks for your review!

I'm ok with all your suggestions except for those I have commented ;)

Tell me what you think about those comments!

File: docs/content/index.md [hunk 4]
```diff
@@ -43,14 +43,14 @@ features:
...
+    title: Display statistics
     details: Calculate and summarize database statistics.
   - icon:
       light: '/database.svg'
       dark: '/database-d.svg'
     title: PostgreSQL, MySQL & SQLite ready
     details:
       Work on top of Doctrine DBAL connections with PostgreSQL, MySQL & SQLite.<br>
-      <small>(Partially works with others database vendors too)</small>
+      <small>(Partially works with other database vendors too.)</small>
```
[the remark this answers is not in the document]
Comment (Lonnytunes) [reply docs/content/index.md:54]: I just removed the "s" at the end of "other" and added a period to the sentence :wink:

[the remark this answers is not in the document]
Comment (SimonMellerin) [reply docs/content/index.md:54]: yes it was an extra thought ;)

File: docs/content/getting-started/introduction.md [hunk 5]
```diff
@@ -10,34 +10,34 @@
...
 </svg>
 
-The idea of this bundle is to make life easier for Symfony developers when managing
-database backup and restoration workflows on one hand, and production data import
-and anonymization on the other.
+The idea behind this bundle is to make life easier for Symfony developers
+in the task of managing database backup and restoration workflows on one hand,
+production data import and anonymization on the other hand.
```
[the remark this answers is not in the document]
Comment (Lonnytunes) [reply docs/content/getting-started/introduction.md:15]: I found that "when managing" was not really appropriate, but it's probably just a question of taste.

File: docs/content/getting-started/installation.md [hunk 6]
```diff
@@ -38,37 +39,33 @@ Then, activate the bundle:
...
 ];
 ```
 
-And copy provided default configuration from vendor:
+Finally, copy the default configuration file from the vendor directory:
 
 ```sh
 cd your_project_dir
-cp vendor/makinacorpus/db-tools-bundle/config/packages/db_tools.yaml config/packages/.
+cp vendor/makinacorpus/db-tools-bundle/config/packages/db_tools.yaml config/packages/
```
[the remark this answers is not in the document]
Comment (Lonnytunes) [reply docs/content/getting-started/installation.md:54]: Why would you want to keep this "." at the end of the command line?

File: docs/content/getting-started/basics.md (hunk 2 above)
Comment (Lonnytunes) [reply docs/content/getting-started/basics.md:50]: I didn't find examples using "doing so" as such, without "by" or "in" ahead, so I decided to change it, but your shorter version is certainly valid as well :slightly_smiling_face:

File: docs/content/getting-started/introduction.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (SimonMellerin) [reply docs/content/getting-started/introduction.md:15]: well re reading this paragraph, i think we should rewrite it completely, the sentence is too long and not fluid to read :/

File: docs/content/getting-started/basics.md (hunk 1 above)
Comment (SimonMellerin) [reply docs/content/getting-started/basics.md:41]: yes i agree, the current version is not so good.

Maybe something like : "Anonymizing production data when using them on your local environment"

File: docs/content/getting-started/basics.md (hunk 2 above)
Comment (Lonnytunes) [reply docs/content/getting-started/basics.md:50]: :ok_hand:

File: docs/content/getting-started/introduction.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Lonnytunes) [reply docs/content/getting-started/introduction.md]: Here is a new shortened version.

**Review by SimonMellerin — APPROVED** [review]

**Pull request merged**